content/en/xtc/xtc-release-notes/v69.md

```diff
@@ -0,0 +1,45 @@
+---
+title: XTC v69
+linkTitle: XTC v69
+
+weight: 147
+type: docs
+
+date: 2023-07-17
+
+description: 2023-07-18
+
+sitemap:
+  changefreq: never
+  priority: 0.1
+---
+
+## Platform
+
+### Project States
+
+The states a project may have in XTC have been refined and enhanced further. In this course, project states have evolved from a simple active/locked marker to a set of different states that influence not only what can be done with the project, but also whether the project is visible at all.
+
+Projects may now have three different states:
+
+* **Active** – The project is visible to all project members. They can act within the project as allowed by their roles. This is the normal state while people work actively on the project.
```

Comment:
As 'act' also has the meaning of 'behave', it might not be right verb here.
```suggestion
* **Active** – The project is visible to all project members. They can interact with the project as allowed by their roles. This is the normal state while people work actively on the project.
```

Reply:
Applied as suggested.